docker-compose/technical/grafana/dashboards/pods_supervision.json

```diff
@@ -1126,13 +1126,213 @@
       ],
       "type": "table"
     },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
uid should be       
```
      "uid": "${datasource}"
```

Reply:
I think you'll get that if you export the dashboard for external use in the share option